app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Recipes</h1>
+<% @recipes.each do |recipe| %>
+<a href="recipes/<%= recipe.id %>">Recipe #<%= recipe.id %></a>
+<div>Name: <%= recipe.name %> by <%= recipe.user.name %></div>
+<div>Description: <%= recipe.description %></div>
+<div>Public? <%= recipe.public %></div>
+<% RecipeFood.where(recipe: recipe).each do | recipe_food | %>
+<div>Total food items: <%= recipe_food.quantity %></div>
+<div>Total price: $<%= recipe_food.quantity * recipe_food.food.price %></div>
+<% end %>
+<% if can? :destroy, recipe %>
+    <%= button_to "REMOVE", "/recipes/#{recipe.id}", method: :delete %>
```

Comment:
- I think it's better not to use the `REMOVE` button for public recipes. The wireframe does not have a button.
<img width="1280" alt="Screen Shot 2022-10-05 at 9 08 50 AM" src="https://user-images.githubusercontent.com/47673311/194057669-bde34b7a-8e6b-4802-8a1c-76b0b6680c04.png">